config/locales/telephony/es.yml

```diff
@@ -7,15 +7,15 @@ es:
       %{app_name} para cancelar.
     authentication_otp:
       sms: |-
-        %{app_name}: Su código de seguridad es %{code}. Este código se vence en %{expiration} minutos. No lo comparta con ninguna persona.
+        %{app_name}: Your one-time code is %{code}. It expires in %{expiration} minutes. Don't share this code with anyone.
```

Comment:
looks like some english slipped in to `es.yml`

Reply:
I was keeping these English to intentionally fail the builds as an indicator of what I still need to get translated. Going to double-check the English rephrasing with UX folks before ordering the translations.

Reply:
Translations added in 1cbdc6f.